daisy_workflows/image_build/windows_for_containers/windows-1803-for-containers.wf.json

```diff
@@ -54,12 +54,12 @@
     "create-image": {
       "CreateImages": [
         {
-          "Name": "windows-server-1709-dc-core-for-containers-v${build_date}",
+          "Name": "windows-server-1803-dc-core-for-containers-v${build_date}",
           "SourceDisk": "${install_disk}",
           "Licenses": ["projects/windows-cloud/global/licenses/windows-for-containers"],
-          "Description": "Microsoft, Windows Server, version 1709 Core for Containers (Beta), Server Core, x64 built on ${build_date}",
+          "Description": "Microsoft, Windows Server, version 1803 Core for Containers, Server Core, x64 built on ${build_date}",
           "GuestOsFeatures": ["VIRTIO_SCSI_MULTIQUEUE", "WINDOWS", "MULTI_IP_SUBNET"],
-          "Family": "windows-1709-core-for-containers",
+          "Family": "windows-1803-core-for-containers",
```

Comment:
Can you add a small comment here that says "the family needs to include the windows version because Windows requires that container images match the base OS version"?

Reply:
Daisy workflows don't have comments, I added a comment to our publish template though